content/en/getting_started/synthetics/api_test.md

```diff
@@ -0,0 +1,62 @@
+## Create an API test
+
+[API tests][1] help you monitor your API endpoints and alert you when they are failing or too slow. These checks verify that your applications are responding to requests and meet any conditions you define, such as response time, HTTP status code, and header or body contents. Use the [Datadog API][2] to see the full list.
+
+In this example, an API test is created to ensure your website is constantly up and providing responses in a specific amount of time.
+
+### Configure the request
+
+1. In the Datadog application, hover over **[UX Monitoring][3]** in the left hand menu and select **Synthetics Test**. 
+2. In the top right corner, click the **New Test** button. 
+3. Select **API test**.
+4. Define the configuration of your API test:
+    - Add the URL of the endpoint you want to monitor. If you don’t know what to start with, you can use https://www.shopist.io/ which is a test web application.”
+    - Select **Advance Options** to use custom request headers, authentication credentials, body content, or cookies.
+    - Add one or two tags, like `prod`, to help organize and filter tests.
+    - Select locations for testing.
+    - Click the **Test URL** button.
+
+#### Define your alert conditions
+
+Now that the test is completed, if you did not create any [assertions][4], they are automatically populated. API tests require at least one assertion to be monitored by Datadog. An assertion is defined by a parameter, an optional property, a comparator, and a target value.
+
+In this example, three default assertions are populated when testing the URL:
+
+{{< img src="getting_started/synthetics/assertions-example.png" alt="Browser test failure" responsive="true" style="width:90%;">}}
+
+These assertions define the alert condition and can be customized. To add a custom assertion, click on any response header in the response preview. You can also click the **New Assertion** button to add an assertion manually, (e.g., `body` contains `Shop.ist`.)
+
+{{< img src="getting_started/synthetics/api-test-configuration.gif" alt="Browser test failure" responsive="true" style="width:90%;">}}
+
+To configure the manual retry of a test when it fails, set:
```

Comment:
Let's mention these two features separately as they're pretty different:


You can set up an alerting condition in order to only be alerted if your endpoint goes down for three minutes, on two different locations:

```
An alert is triggered if your test fails for 3 minutes from any 2 of 13 locations`
```

You can also decide after how many failures you want a location to be considered as fail using the below option:

```
Retry x time before location is marked as failed
```